apps/yapms/src/routes/app/[country]/[map]/[year]/initialize/ApplyPanZoom.ts

```diff
@@ -1,16 +1,25 @@
 import panzoom, { type PanZoom } from 'panzoom';
 import z from 'zod';
 
+let originalBounds: DOMRect;
+let newBounds: DOMRect;
+
 function applyPanZoom(mapBind: HTMLDivElement): PanZoom | undefined {
 	const svg = mapBind.querySelector<SVGElement>('svg'); //We can do this since the only child of node is the map svg itself.
 	if (svg) {
+		if (svg.classList.contains('insetsHidden')) {
+			boundsNoInset(svg);
+		}
 		const panzoomInstance = panzoom(svg, {
 			minZoom: 0.5,
 			maxZoom: 100,
 			smoothScroll: false,
 			autocenter: true,
 			zoomDoubleClickSpeed: 1
 		});
+		if (svg.classList.contains('insetsHidden')) {
+			boundsInset(svg);
+		}
```

Comment:
We need bounds to be set to no insets for panzoom to zoom in on regions specifically, but then we need them to be set back to include insets because if the user turns on insets, they will be cut off if bounds are not restored to the original. 

![image](https://github.com/yapms/yapms/assets/42476312/5e095b4a-f0b3-47c4-96d8-954427148d7e)